restreamer/restreamer/main.py

```diff
@@ -493,23 +480,43 @@ def get_frame(channel, quality):
 
 @app.route('/thumbnail/<channel>/<quality>.png')
 @request_stats
-@has_path_args
```

Comment:
This function still has path args (`channel` and `quality`) which need to be checked by `has_path_args` for security.